src/ToggleBadge.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { theme as themeGet, space, fontSize, propTypes } from 'styled-system'
+import theme from './theme'
+
+const ToggleBadge = styled.div`
```

Comment:
Unsure of how this is intended to be used, but if this is meant to be a clickable element, it should be a `<button>` element

Reply:
I think this is a blocker unless this element is *not* meant to be clicked